Dockerfile

```diff
@@ -5,12 +5,10 @@ ENV UWSGI_HTTP_AUTO_CHUNKED=1 UWSGI_HTTP_KEEPALIVE=1 UWSGI_LAZY_APPS=1 UWSGI_WSG
 
 ## System preparation steps ################################################# ##
 
-# We do the whole installation and configuration in one layer:
+RUN zypper -n in gcc libffi-devel && zypper clean && rm -rf /var/cache
+
 COPY requirements.txt /pcw/
-# * Install system requirements
-# * Install pip requirements
-# * Empty system cache to conserve some space
-RUN zypper -n in gcc libffi-devel && pip install --no-cache-dir -r /pcw/requirements.txt && zypper clean && rm -rf /var/cache
+RUN pip install --no-cache-dir -r /pcw/requirements.txt
```

Comment:
this will dramatically increase overall weight of container because you basically adding extra layer where we had one . What the benefit of that ?

Reply:
> this will dramatically increase overall weight of container because you basically adding extra layer where we had one . What the benefit of that ?

Size increased by only 90MB but I suspect it came mostly from updated dependencies.

The benefit is that I don't have to wait for OS requirements to be installed when just updating requirements.txt.